app/reducers/loginError.jsx

```diff
@@ -0,0 +1,22 @@
+
+const SET_ERROR = 'SET_ERROR'
```

Comment:
Can we add an extra line here? Just for readability.

(Actually, I'd prefer if we divided these with the like /* ----- WORDS HERE ----- */ things, but we can go back and mass add those in later.)